Gemfile

```diff
@@ -7,6 +7,7 @@ gem "bootstrap-sass", "3.3.7"
 gem "bootstrap-will_paginate", "1.0.0"
 gem "config"
 gem "faker", "1.7.3"
+gem "figaro"
```

Comment:
gem này dùng để làm gì em

Reply:
VÌ khi thực hiện gửi email, mình cần phải lưu username và pass để ứng dụng có thể truy cập vào máy chủ SMTP để gửi email. Những lưu trữ nhạy cảm này sẽ nằm trong các biến môi trường và mình không muốn thông tin đó được hiển thị lên git. Do đó cần phải dùng gem figaro này để cấu hình các ứng dụng an toàn hơn trong rails